src/components/Heart/Heart.css

```diff
@@ -1,31 +1,40 @@
-.large-font {
-    font-size: 50px;
-    border: none;
-    background-color: white;
+.heart {
+  font-size: 40px;
+  fill: rgb(137, 137, 137);
+  cursor: pointer;
+  transition: all 0.5s;
+}
+
+.heart.active {
+  font-size: 40px;
+  fill: rgb(225, 46, 46);
+  animation: like 0.5s 1;
+}
+
+.heart:hover {
+  transform: scale(1.1);
+}
+
+@keyframes like {
+  0% {
+    transform: scale(1);
   }
-  
-  .heart.active {
-    animation: like 0.5s 1;
-    fill: red;
+  90% {
+    transform: scale(1.2);
   }
-  
-  .heart {
-    transition: all 0.5s;
-    fill: grey;
-  
+  100% {
+    transform: scale(1.1);
   }
-  
-  
-  @-webkit-keyframes like {
-    0% {
-      transform: scale(1);
-    }
-  
-    90% {
-      transform: scale(1.2);
-    }
-  
-    100% {
-      transform: scale(1.1);
-    }
-  }
\ No newline at end of file
+}
```

Comment:
The @-webkit-keyframes lines 30 - 40 can be deleted because lines 18 -28 replaces the need for the @-webkit-keyframes.